bigtable/integration_test.go

```diff
@@ -127,16 +133,27 @@ func cleanup(c IntegrationTestConfig) error {
 		return err
 	}
 
-	for _, info := range instances {
-		if strings.HasPrefix(info.Name, prefixOfInstanceResources) {
-			timestamp := info.Name[len(prefixOfInstanceResources):]
+	for _, instanceInfo := range instances {
+		if strings.HasPrefix(instanceInfo.Name, prefixOfInstanceResources) {
+			timestamp := instanceInfo.Name[len(prefixOfInstanceResources):]
 			t, err := strconv.ParseInt(timestamp, 10, 64)
 			if err != nil {
 				return err
 			}
 			uT := time.Unix(t, 0)
 			if time.Now().After(uT.Add(timeUntilResourceCleanup)) {
-				iac.DeleteInstance(ctx, info.Name)
+				iac.DeleteInstance(ctx, instanceInfo.Name)
+			}
+		} else {
+			// Delete clusters created in existing instances
+			clusters, err := iac.Clusters(ctx, instanceInfo.Name)
```

Comment:
to confirm, is there ever a case where we want to keep the instance *and* cluster?

Reply:
We don't want to keep the ones created by the integration test.

The ones created by user will not be deleted.
User can specify an existing instance and a existing cluster to use in tests via environment variables:
GCLOUD_TESTS_BIGTABLE_INSTANCE
GCLOUD_TESTS_BIGTABLE_CLUSTER

There are prefix checks at line 137 and 154 to determine whether resource was created by the test